scripts/cram

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+# if we have cram already, use that
```

Comment:
okay, to review, I actually fallback to system cram if it's available. I dunno what you guys think about this